### DISCUSSION TITLE
Dsc alexor PC9155

### DISCUSSION THREAD

**Discussion by rdaguerre** [taligentx/dscKeybusInterface#128] [dscKeybusInterface Discussion]
Hi, where can i connect the arduino in this panel? . He have pclink (serial port). Thanks very much

**Comment by taligentx** [comment]
Hi @rdaguerre - late response, the DSC Alexor isn't supported as it does not use the Keybus protocol.  The PC-Link interface is a separate protocol and would be a good project if someone is up for decoding the protocol and emulating the DSC DLS software but is out scope for this project.
